CODEOWNERS

```diff
@@ -34,3 +34,12 @@
 /examples/sql-database-projects/_ @ErikEJ @jmezach
 /src/CommunityToolkit.Aspire.Hosting.SqlDatabaseProjects/_ @ErikEJ @jmezach
 /tests/CommunityToolkit.Aspire.Hosting.SqlDatabaseProjects/_ @ErikEJ @jmezach
+
+# CommunityToolkit.Aspire.SurrealDb
+# CommunityToolkit.Aspire.Hosting.SurrealDb
+
+/examples/surrealdb/_ @Odonno
+/src/CommunityToolkit.Aspire.Hosting.SurrealDb/_ @Odonno
+/tests/CommunityToolkit.Aspire.Hosting.SurrealDb.Tests/_ @Odonno
+/src/CommunityToolkit.Aspire.SurrealDb/_ @Odonno
+/tests/CommunityToolkit.Aspire.SurrealDb.Tests/_ @Odonno
```

Comment:
For the CODEOWNERS to be valid you would need to join the maintainer group - is that something you're interested in (I'm guessing so since you put yourself down as the owner 😉)

Reply:
Why not.

Reply:
That should be good now. Can you reopen the PR @aaronpowell ?

Reply:
@aaronpowell ?